src/scss/components/_methodology.scss

```diff
@@ -0,0 +1,65 @@
+@use 'abstracts' as *;
+
+.methodology {
+  position: relative;
+  display: grid;
+  grid-column-gap: rem(72px);
+  grid-row-gap: rem(16px);
+  padding: rem(48px) 0;
+  
+  @include breakpoint('medium') {
+    grid-template-columns: 2fr 1fr;
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: $color__gray-700;
+  }
+
+
+  &__title {
+    grid-column: 1 / span 2;
+    margin-top: 0;
+    margin-bottom: 0;
+    color: $color__white;
+    font-family: $font__expo;
+    @include font-size(34px);
+    line-height: 1;
+  }
+  
+  &__desc {
+    margin-bottom: 0;
+    color: $color__white-190;
+    @extend %body__text-l;
+    
+  }
+  
+  &__program-desc {
+    display: none;
```

Comment:
This actually should be visible on mobile. Leaving it off was a mistake on Tucker's part. Once you add it back in, you'll want to move the padding-left property to be inside the media query.